akd/agents/gap_analysis/prompts.py

```diff
@@ -0,0 +1,259 @@
+from langchain_core.prompts import ChatPromptTemplate
+
+# =============================================================================
+# Groups scientific paper section titles into hierarchical section clusters
+# =============================================================================
+
+section_grouper_inst = """I have a structured list of section titles from a scientific paper, and I want you to group them into logical sections. Each group should represent a distinct part of the document, starting with the title, followed by related subsections if present. Here's an example:
+
+Input Example:
+['KnowledgeHub: An End-to-End Tool for Assisted Scientific Discovery',
+'Abstract',
+'1 Introduction',
+'2 System Description',
+'2.1 Document Ingestion',
+'2.2 Annotation',
+'2.3 Question Answering',
+'3 Use-case: Knowledge Discovery for the Battery Domain',
+'4 Conclusion',
+'Ethical Statement',
+'Acknowledgments',
+'References']
+
+Output Example:
+[['KnowledgeHub: An End-to-End Tool for Assisted Scientific Discovery'], ['Abstract'], ['1 Introduction'], ['2 System Description', '2.1 Document Ingestion', '2.2 Annotation', '2.3 Question Answering'], ['3 Use-case: Knowledge Discovery for the Battery Domain'], ['4 Conclusion'], ['Ethical Statement'], ['Acknowledgments'], ['References']]
+
+Make sure to preserve the hierarchy of sections and subsections. ONLY provide the output.
+Now, group the following lists in the same way."""
+
+
+section_grouper_prompt = ChatPromptTemplate.from_messages(
```

Comment:
Can we make this all CAPSLOCK to conform to global variable naming convention?
`SECTION_GROUPER_PROMPT`